tools/msvs/msi/nodemsi.wixproj

```diff
@@ -78,7 +78,9 @@
     <!--
     <PostBuildEvent>
       "$(WixToolPath)\torch.exe" -t language "$(TargetDir)en-US\$(TargetFileName)" "$(TargetDir)de-de\$(TargetFileName)" -out "$(TargetDir)transforms\de-de.mst"
+      "$(WixToolPath)\torch.exe" -t language "$(TargetDir)en-US\$(TargetFileName)" "$(TargetDir)it-it\$(TargetFileName)" -out "$(TargetDir)transforms\it-it.mst"
```

Comment:
don't forget to add also the it-it.wxl to the project.

You should add at line 58:
 <EmbeddedResource Include="i18n\it-it.wxl" />